addons/purchase/tests/test_purchase.py

```diff
@@ -22,76 +22,6 @@ def setUpClass(cls):
         cls.vendor = cls.env['res.partner'].create({'name': 'vendor1'})
         cls.uom_unit = cls.env.ref('uom.product_uom_unit')
 
-    def test_date_planned_1(self):
```

Comment:
Could be replaced by a simple test on your compute: 
Create a purchase order with some lines. Check the expected_date = min(line dates)

Reply:
Replaced test_date_planned_1 with a simpler test for checking expected_date. Hopefully it's not too long.